README.md

```diff
@@ -73,7 +74,7 @@ Experiment with flagd in your browser using [the Killercoda tutorial](https://ki
       --uri file:./example_flags.flagd.json
     ```
 
-    Or use docker:
+    Or use docker (!! this does not work on Windows without using WSL !!):
```

Comment:
Moving this to italic text & in a new line with a `Note`

And rather than saying `does not work`, reword to say something like "In Windows, this requires WSL"